src/anderson_darling.jl

```diff
@@ -89,99 +114,192 @@ the samples come from different distributions.
 `modified` parameter enables a modified test calculation for samples whose observations
 do not all coincide.
 
+`nsim` parameter if it equals to 0 then the asymptotic calculation of p-value is used.
```

Comment:
Still not correct grammar. Something like "If `nsim` is equal to 0 (the default), the asymptotic... If it is greater than 0, an estimation... is used. This proportion is...".